modules/persistence/src/services/metadata/ToC/utils/convertSlugToUrl.ts

```diff
@@ -0,0 +1,11 @@
+export const convertSlugToUrl = (slug, projectSlug, prefix, url) => {
+  return ensureTrailingSlash(url) + ensureTrailingSlash(prefix) + projectSlug + ensureLeadingSlash(slug);
```

Comment:
does the prefix not already contain the projectSlug?

ie. 
![image](https://user-images.githubusercontent.com/13054820/214136573-93a6ce0c-e134-47fb-bafb-291c03940cc9.png)

(also on minor note above looks off? prefix.dotcomstg for pool.repos_branches )

Reply:
🤦‍♂️ very good catch